### PR TITLE
#1395 Add a fromVersion and a toVersion option in a JaVers Query

### PR DESCRIPTION
**Pull request by AnthonyInMath**

### CODE REVIEW

File: javers-core/src/main/java/org/javers/repository/jql/QueryBuilder.java [hunk 1]
```diff
@@ -508,13 +508,38 @@ public QueryBuilder withCommitPropertyLike(String name, String valueFragment){
...
+        queryParamsBuilder.fromVersion(fromVersion);
+        return this;
+    }
+
+    /**
+     * All snapshots up to (exclusive) a given version.
+     * @see #withVersion(long)
+     * @see #fromVersion(long)
+     */
+    public QueryBuilder toVersion(long toVersion) {
```
Comment (bartoszwalacik) [inline javers-core/src/main/java/org/javers/repository/jql/QueryBuilder.java:537]: Making fromVersion inclusive but toVersion exclusive, seems counter intuitive for me. Could you please change toVersion also to be inclusive (so from to would work like the BETWEEN operator in SQL)

**Comment by bartoszwalacik** [comment]
new feature but no new test :) please add a new test case(s) in `JaversRepositoryE2ETest`
No more comments from me, the impl LGTM

**Comment by AnthonyInMath** [comment]
Done :)

**Comment by bartoszwalacik** [comment]
thanks, LGTM, I have added just a few style fixes, the PR will be merged from here https://github.com/javers/javers/pull/1398/commits

**Comment by bartoszwalacik** [comment]
thanks @AnthonyInMath 
Merged from this PR https://github.com/javers/javers/pull/1398. I will release it soon

**Pull request closed without merging**